### ISSUE TITLE
Upgrade from C116 -> C117 on Win x64 via 1.59.x - Nightly (Checking C117)

### ISSUE THREAD

**Issue by LaurenWags** [brave/brave-browser#32669]
As per process, QA runs through the following cases to ensure that the major chromium bump that's about to get merged into `master` doesn't regress the `Nightly` channel. Once the major chromium bump is merged into `master`, QA will run through a full manual pass on `Windows` & `Android`.

### Startup & Components

- [x] Verify that Brave is only contacting `*.brave.com` endpoints on first launch using either `Charles Proxy`, `Fiddler`, `Wireshark` or `LittleSnitch` (or a similar application)
  - [x] Verify that opening a NTP doesn't trigger any outbound connections related to widgets without user interaction
- [x] Remove the following component folders and ensure that they're being re-downloaded after restarting the browser:
  - [x] `afalakplffnnnlkncjhbmahjfjhmlkal`: `AutoplayWhitelist.dat`, `ExtensionWhitelist.dat`, `ReferrerWhitelist.json` and `Greaselion.json`
  - [x] `CertificateRevocation`
  - [x] `gccbbckogglekeggclmmekihdgdpdgoe`: (Sponsored New Tab Images)
  - [x] `gkboaolpopklhgplhaaiboijnklogmbc`: Brave Ad Block List Catalog
  - [x] `iodkpdagapdfkphljnddpjlldadblomo`: Brave Ad Block Updater
  - [x] `mfddibmblmbccpadfndgakiopmmhebop`: Brave Ad Block List Catalog
  - [x] `oofiananboodjbbmdelgdommihjbkfag`: HTTPSE
  - [x] `Safe Browsing`
- [x] Restart the browser, load `brave://components`, wait for 8 mins and verify that no component shows any errors

**Note:** Always double check `brave://components` to make sure there's no errors/missing version numbers

### Upgrade

- [x] Make sure that data from the last version appears in the new version OK
- [x] Ensure that `brave://version` lists the expected Brave & Chromium versions
- [x] With data from the last version, verify that:
  - [x] Bookmarks on the bookmark toolbar and bookmark folders can be opened
  - [x] Cookies are preserved
  - [x] Installed extensions are retained and work correctly
  - [x] Opened tabs can be reloaded
  - [x] Stored passwords are preserved
  - [x] Sync chain created in previous version is retained
  - [x] Social media-blocking buttons changes are retained
  - [x] Custom filters under brave://settings/shields/filters are retained
  - [x] Custom lists under brave://settings/shields/filters are retained
  - [ ] Rewards
    - [x] BAT balance is retained
    - [x] Auto-contribute list is retained
    - [x] Both Tips and Monthly Contributions are retained
    - [x] Panel transactions list is retained
    - [x] Changes to rewards settings are retained
    - [x] Ensure that Auto Contribute is not being enabled when upgrading to a new version if AC was disabled
  - [x] Ads
    - [x] Both `Estimated pending rewards` & `Ad notifications received this month` are retained
    - [x] Changes to ads settings are retained
    - [x] Ensure that ads are not being enabled when upgrading to a new version if they were disabled
    - [x] Ensure that ads are not disabled when upgrading to a new version if they were enabled

**Comment by MadhaviSeelam** [comment]
Verification `PASSED` using
```
Brave | 1.59.62 Chromium: 117.0.5938.35 (Official Build) nightly (64-bit)
-- | --
Revision | f40314adc0750775c96f308eb85fd6614978dac6
OS | Windows 11 Version 22H2 (Build 22621.2134)
```

**Comment by kjozwiak** [comment]
Verification **PASSED** on `Win 10 x64` using the following build(s):

```
Brave | 1.59.66 Chromium: 117.0.5938.35 (Official Build) nightly (64-bit)
-- | --
Revision | 344cc1226dacfaff54fcd88330a96a88af3efca2
OS | Windows 10 Version 22H2 (Build 19045.3393)
```

<details>
<summary>C116 --> C117 on-demand upgrade check - PASSED</summary>

* installed `1.59.70 Chromium: 116.0.5845.163` and closed Brave
* created `HKEY_LOCAL_MACHINE\SOFTWARE\WOW6432Node\BraveSoftware\UpdateDev\url`
  * set `url` as `https://updates.brave.com/service/update2`
* set `HKEY_LOCAL_MACHINE\SOFTWARE\WOW6432Node\BraveSoftware\Update\ClientState\{C6CB981E-DB30-4876-8639-109F8933582C}\ap` as `test-nite`
* realunched Brave and visited `brave://settings/help` and ensured that new update was being pulled/downloaded
* ensured that clicking `Relaunch` restarted the browser without any issues
* ensured Brave was updated to `1.59.66 Chromium: 117.0.5938.35` without any issues and you can restart the browser

**`Example`** | **`Example`** | **`Example`** | **`Example`** | **`Example`**
----------------|----------------|----------------|----------------|------------------
![image](https://github.com/brave/brave-browser/assets/2602313/0bf7802f-9080-44df-867e-5283d48a5163) | ![image](https://github.com/brave/brave-browser/assets/2602313/86d75c1f-eabb-4c56-af8e-abe7cde8ed89) | ![image](https://github.com/brave/brave-browser/assets/2602313/f478eb09-0ed3-4e45-ac8b-74e0de9fe0bb) | ![image](https://github.com/brave/brave-browser/assets/2602313/969b20d4-39ea-4acc-b866-33f8d76937a0) | ![image](https://github.com/brave/brave-browser/assets/2602313/fb0ae072-745d-4b94-aa5a-56a706032587)
</details>

<details>
<summary>C117 --> C117 on-demand upgrade check - PASSED</summary>

* installed `1.59.62 Chromium: 117.0.5938.35` via `BraveBrowserStandaloneNightlySetup_117_1_59_62.exe` and closed Brave
* created `HKEY_LOCAL_MACHINE\SOFTWARE\WOW6432Node\BraveSoftware\UpdateDev\url`
  * set `url` as `https://updates.brave.com/service/update2`
* set `HKEY_LOCAL_MACHINE\SOFTWARE\WOW6432Node\BraveSoftware\Update\ClientState\{C6CB981E-DB30-4876-8639-109F8933582C}\ap` as `test-nite`
* realunched Brave and visited `brave://settings/help` and ensured that new update was being pulled/downloaded
* ensured that clicking `Relaunch` restarted the browser without any issues
* ensured Brave was updated to `1.59.66 Chromium: 117.0.5938.35` without any issues and you can restart the browser

**`Example`** | **`Example`** | **`Example`** | **`Example`** | **`Example`**
----------------|----------------|----------------|----------------|------------------
![image](https://github.com/brave/brave-browser/assets/2602313/c15dc550-fdd2-4a02-ad08-d09d666e2fbf) | ![image](https://github.com/brave/brave-browser/assets/2602313/c434fe9c-4e7a-4da8-89ec-017ad218dba0) | ![image](https://github.com/brave/brave-browser/assets/2602313/e9a3b758-cb0f-4605-9275-719792053918) | ![image](https://github.com/brave/brave-browser/assets/2602313/ca1449b8-5caa-47d4-9e15-c2d6c0d8c18f) | ![image](https://github.com/brave/brave-browser/assets/2602313/40d03f47-43b0-4c99-88e1-2d7c2b73c2a5)

**Comment by kjozwiak** [comment]
Verification **PASSED** on `Win 10 x86` using the following build(s):

```
Brave | 1.59.66 Chromium: 117.0.5938.35 (Official Build) nightly (32-bit)
-- | --
Revision | 344cc1226dacfaff54fcd88330a96a88af3efca2
OS | Windows 10 Version 22H2 (Build 19045.3393)
```

<details>
<summary>C116 --> C117 on-demand upgrade check - PASSED</summary>

* installed `1.59.70 Chromium: 116.0.5845.163` and closed Brave
* created `HKEY_LOCAL_MACHINE\SOFTWARE\BraveSoftware\UpdateDev\url`
  * set `url` as `https://updates.brave.com/service/update2`
* set `HKEY_LOCAL_MACHINE\SOFTWARE\BraveSoftware\Update\ClientState\{C6CB981E-DB30-4876-8639-109F8933582C}\ap` as `86-ni-test`
* realunched Brave and visited `brave://settings/help` and ensured that new update was being pulled/downloaded
* ensured that clicking `Relaunch` restarted the browser without any issues
* ensured Brave was updated to `1.59.66 Chromium: 117.0.5938.35` without any issues and you can restart the browser

**`Example`** | **`Example`** | **`Example`** | **`Example`** | **`Example`**
----------------|----------------|----------------|----------------|------------------
![image](https://github.com/brave/brave-browser/assets/2602313/079cdffa-c1fc-40d9-bd44-bc82bd64a73b) | ![image](https://github.com/brave/brave-browser/assets/2602313/644fddba-7d17-47eb-92d5-1a03d8b16883) | ![image](https://github.com/brave/brave-browser/assets/2602313/93150348-01e7-4d86-b4ea-5e416a828a82) | ![image](https://github.com/brave/brave-browser/assets/2602313/2bb44748-dfc0-4be3-9a43-f90f7d9660e4) | ![image](https://github.com/brave/brave-browser/assets/2602313/fd9e49e0-80ab-4140-9c42-5cf160b88144)
